translations-script/import.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const _ = require('lodash');
+const readXlsxFile = require('read-excel-file/node');
+
+async function main(filename) {
+  const en = {};
+  const ga = {};
+
+  const [, ...rows] = await readXlsxFile(filename);
+  rows.forEach((row) => {
+    const path = row[0];
+    _.set(en, path, row[1]);
+    _.set(ga, path, row[2]);
+  });
+
+  return [en, ga];
+}
+
+main('input.xlsx').then(([en, ga]) => {
+  fs.writeFileSync('en.json', JSON.stringify(en, null, 2));
+  fs.writeFileSync('ga.json', JSON.stringify(ga, null, 2));
+});
```

Comment:
extremely minor, but these could also be replaced with `fs.promises.writeFile()` but it's fine without.

Reply:
I did think that but I wasn't 100% sure 👍 
Thank you!